frontend/src/js/small-tab-navigation/SmallTabNavigation.tsx

```diff
@@ -26,7 +26,7 @@ const SmallTabNavigation: FC<PropsT> = ({
   onSelectTab,
 }) => {
   return (
-    <div className={className}>
+    <div className={className} style={{"display": "flex"}}>
```

Comment:
What's the reason for that change? The parent components are currently setting layout settings like this. Plus, if we style, let's please try and avoid inline styles and use styled components instead

Reply:
![image](https://user-images.githubusercontent.com/118362236/204755231-d1f22548-5411-4918-81dd-1a77a3ecd9fa.png)
Breaks due to the Div in HoverNavigatable. But I will remove the style here